libcassowary/src/unit/MobileUnitTestEngine.php

```diff
@@ -399,6 +402,23 @@ private function parseiOSOutput($test_results) {
         return $result_array;
     }
 
+	// Retrieve Args from the xctool-args file
+	private function xcodebuild_args() {
+		$xctoolargs_path = $this->projectRoot . "/.xctool-args";
```

Comment:
This appears to be a required file which isn't always true either. Needs to have some fallback support.